### DISCUSSION TITLE
Maybe `return` statements are missing in podman/pkg/domain/infra/abi/containers.go ?

### DISCUSSION THREAD

**Discussion by eriksjolund** [containers/podman#22377] [Q&A] [answered]
In this for loop
https://github.com/containers/podman/blob/b8a684b64d77fc97b3b539dd5891319756c15c39/pkg/domain/infra/abi/containers.go#L1113-L1125

It looks like one iteration could set `report.ExitCode` like this

```
report.ExitCode = 126
```
and possibly the next iteration could  set `report.ExitCode` like this

```
report.ExitCode = 0
```

At first sight it looks like that there are `return` statements missing inside the loop.

I don't know this code so I might be wrong.
What do you think?

**Comment by Luap99** [comment, marked as answer]
responses should only ever be length 1 here as we only pass one name to `ic.ContainerRestore()`  so it should not be problem, although yeah the code never enforces that assumption.

**Reply by eriksjolund** [reply]
Thanks for clarifying!
